oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -389,7 +402,7 @@ public APIGatewayProxyResponseEvent handleRequest(
                 } else {
                     sessionService.storeOrUpdateSession(userSession.setNewAccount(accountState));
                 }
-
+                orchSessionService.updateSession(orchSession.withAccountState(orchAccountState));
```

Comment:
Is this meant to be feature flagged?

Reply:
Nope as we hit this lambda running through a journey in dev so I can be certain this lambda works